tbdex/rfq_create.go

```diff
@@ -0,0 +1,226 @@
+package tbdex
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"time"
+
+	"go.jetpack.io/typeid"
+)
+
+// CreateRFQ creates an [RFQ]
+//
+// # An RFQ is a resource created by a customer of the PFI to request a quote
+//
+// [RFQ]: https://github.com/TBD54566975/tbdex/tree/main/specs/protocol#rfq-request-for-quote
+func CreateRFQ(from, to, offeringID string, payin PayinMethodWithPrivate, payout PayoutMethodWithPrivate, opts ...CreateRFQOption) (RFQ, error) {
+	defaultID, err := typeid.WithPrefix(RFQKind)
+	if err != nil {
+		return RFQ{}, fmt.Errorf("failed to generate default id: %w", err)
+	}
+
+	r := createRFQOptions{
+		id:         defaultID.String(),
```

Comment:
```suggestion
	r := createRFQOptions{
		id:         typeid.Must(typeid.WithPrefix(RFQKind)),
```

can do this because it guaranteed won't throw an error given that we know the prefix we're passing in is valid